README.md

```diff
@@ -52,15 +52,26 @@ defmodule MyApp.Guardian do
   use Guardian, otp_app: :my_app
 
   def subject_for_token(resource, _claims) do
-    {:ok, to_string(resource.id)}
+    # You could send any information you want
+    # as long as it will be useful to find the resource back
+    # from the token `claims["sub"]` on `resource_from_claims` function
+    sub = to_string(resource.id)
+    {:ok, sub}
   end
-
   def subject_for_token(_, _) do
     {:error, :reason_for_error}
   end
 
   def resource_from_claims(claims) do
-    {:ok, find_me_a_resource(claims["sub"])}
+    # In this case `claims["sub"]` is the `id of the resource`
+    # because we previously used the `id` of the resource for `sub`
+    # of the tokens see `subject_for_token` function.
+    id = claims["sub"]
+    resource = MyApp.get_resource_by_id(id)
+
+    # Do whatever you want for find the resource
```

Comment:
Let's remove this comment.  I think this is applied in the previous comments.

Reply:
@doomspork at this point I am commenting for kids (no offenses but people don't like to read and study anymore, they just want to copy + paste)